Libs/MRML/Core/vtkMRMLDisplayNode.h

```diff
@@ -252,17 +278,29 @@ class VTK_MRML_EXPORT vtkMRMLDisplayNode : public vtkMRMLNode
   vtkBooleanMacro(Clipping, int);
 
   /// Set the slice intersection visibility of the display node.
-  /// \sa SliceIntersectionVisibility, GetSliceIntersectionVisibility(),
+  /// Function to manage \sa Visibility2D for backwards compatibility
+  /// \sa Visibility2D, GetSliceIntersectionVisibility(),
   /// SliceIntersectionVisibilityOn(), SliceIntersectionVisibilityOff()
-  vtkSetMacro(SliceIntersectionVisibility, int);
+  /// \deprecated SetSliceIntersectionVisibility
+  void SetSliceIntersectionVisibility(int on);
   /// Get the slice intersection visibility of the display node.
-  /// \sa SliceIntersectionVisibility, SetSliceIntersectionVisibility(),
+  /// Function to manage \sa Visibility2D for backwards compatibility
+  /// \sa Visibility2D, SetSliceIntersectionVisibility(),
   /// SliceIntersectionVisibilityOn(), SliceIntersectionVisibilityOff()
-  vtkGetMacro(SliceIntersectionVisibility, int);
+  /// \deprecated GetSliceIntersectionVisibility
+  int GetSliceIntersectionVisibility();
+  /// Set the slice intersection visibility of the display node.
+  /// Function to manage \sa Visibility2D for backwards compatibility
+  /// \sa Visibility2D, SetSliceIntersectionVisibility(),
```

Comment:
Don't think `Visibility2D` visibility will link to the private member.

Reply:
I left this in because I'm curious whether it is properly referenced or not. I don't see any good reason why not (private members are also listed in doxygen), and it's useful to link to those. I'll remove it later if they are not link.